### PR TITLE
Use multi-arch etcd image

### LINKED ISSUE #76134: Use the multi-arch manifest etcd for the e2e conformance tests
**What happened**:
I see only amd64 architecture etcd is used for the [conformance](https://github.com/kubernetes/kubernetes/blob/master/test/utils/image/manifest.go#L63) which will create issues for the other architectures like ppc64le, arm etc. We need to replace this etcd image with the k8s maintained fat manifest image present in `gcr.io/google-containers/etcd:3.3.10`

### PR DESCRIPTION
**Pull request by mkumatag**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change

/kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:
**Which issue(s) this PR fixes**:

Fixes: https://github.com/kubernetes/kubernetes/issues/76134

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by mkumatag** [comment]
Seems like I need some more changes to use the k8s etcd image, moving this task to WIP

**Comment by mkumatag** [comment]
Now this PR is ready for the review, e2e tests passed but timedout while teardown.. will invoke one more time and see how it goes..

**Comment by mkumatag** [comment]
Now its all green can someone give lgtm and approve? @ixdy @luxas @listx @cblecker

File: test/utils/image/manifest.go [hunk 1]
```diff
@@ -28,7 +28,6 @@ import (
 type RegistryList struct {
 	DockerLibraryRegistry string `yaml:"dockerLibraryRegistry"`
 	E2eRegistry           string `yaml:"e2eRegistry"`
-	EtcdRegistry          string `yaml:"etcdRegistry"`
 	GcRegistry            string `yaml:"gcRegistry"`
 	PrivateRegistry       string `yaml:"privateRegistry"`
 	SampleRegistry        string `yaml:"sampleRegistry"`
```
Comment (BenTheElder) [inline test/utils/image/manifest.go]: not a blocker for this PR: I would love to see these commented at some point.

Comment (mkumatag) [reply test/utils/image/manifest.go]: ack

**Review by BenTheElder — APPROVED** [review]
/lgtm
/approve

**Pull request merged**